test/include/input.njk

```diff
@@ -0,0 +1 @@
+{% include "includes/child.njk" %}
```

Comment:
Path should be `{% include "include/child.njk" %}`, then the test succeeds. Still checking why it's not working for me in the situation as filed in issue #10